run_tests.py

```diff
@@ -20,11 +22,13 @@
     # Use the workbench settings file:
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "workbench.settings")
     # Configure a range of ports in case the default port of 8081 is in use
-    os.environ.setdefault("DJANGO_LIVE_TEST_SERVER_ADDRESS", "localhost:8081-8099")
+    # os.environ.setdefault("DJANGO_LIVE_TEST_SERVER_ADDRESS", "localhost:8081-8099")
 
     # Silence too verbose Django logging
     logging.disable(logging.DEBUG)
 
+    settings.INSTALLED_APPS.append('problem_builder')
```

Comment:
Instead of doing this, please create a new settings file. That way it will be possible to run other commands with that context.